primevue/api: TypeScript compilation errors due to wrong declarations

Currently, a couple of type declarations in `src/components/api/Api.d.ts` https://github.com/primefaces/primevue/blob/e7694951814f33a10c8a18c51c3392a6e53a4427/src/components/api/Api.d.ts#L1-L23 are mistakenly written as assignments, leading to a host of TypeScript compilation errors when `primevue/api` is imported:

```console
$ tsc src/components/api/Api.d.ts
src/components/api/Api.d.ts(1,32): error TS1254: A 'const' initializer in an ambient context must be a string or numeric literal or literal enum reference.
src/components/api/Api.d.ts(2,16): error TS1162: An object member cannot be declared optional.
src/components/api/Api.d.ts(2,19): error TS2693: 'string' only refers to a type, but is being used as a value here.
src/components/api/Api.d.ts(3,13): error TS1162: An object member cannot be declared optional.
src/components/api/Api.d.ts(3,16): error TS2693: 'string' only refers to a type, but is being used as a value here.
src/components/api/Api.d.ts(4,17): error TS1162: An object member cannot be declared optional.
src/components/api/Api.d.ts(4,20): error TS2693: 'string' only refers to a type, but is being used as a value here.
src/components/api/Api.d.ts(5,14): error TS1162: An object member cannot be declared optional.
src/components/api/Api.d.ts(5,17): error TS2693: 'string' only refers to a type, but is being used as a value here.
src/components/api/Api.d.ts(6,11): error TS1162: An object member cannot be declared optional.
src/components/api/Api.d.ts(6,14): error TS2693: 'string' only refers to a type, but is being used as a value here.
src/components/api/Api.d.ts(7,15): error TS1162: An object member cannot be declared optional.
src/components/api/Api.d.ts(7,18): error TS2693: 'string' only refers to a type, but is being used as a value here.
src/components/api/Api.d.ts(8,7): error TS1162: An object member cannot be declared optional.
src/components/api/Api.d.ts(8,10): error TS2693: 'string' only refers to a type, but is being used as a value here.
src/components/api/Api.d.ts(9,14): error TS1162: An object member cannot be declared optional.
src/components/api/Api.d.ts(9,17): error TS2693: 'string' only refers to a type, but is being used as a value here.
src/components/api/Api.d.ts(10,26): error TS1162: An object member cannot be declared optional.
...
```

--- a/src/components/api/Api.d.ts
+++ b/src/components/api/Api.d.ts
@@ -1,4 +1,4 @@
-export const FilterMatchMode = {
+export const FilterMatchMode: {
     STARTS_WITH?: string,
     CONTAINS?: string,
     NOT_CONTAINS?: string,
@@ -17,7 +17,7 @@ export const FilterMatchMode = {
     DATE_AFTER?: string
 }
 
-export const FilterOperator = {
+export const FilterOperator: {
     AND?: string,
     OR?: string
 }
@@ -49,7 +49,7 @@ interface FilterServiceOptions {
 
 export const FilterService: FilterServiceOptions;
 
-export const PrimeIcons = {
+export const PrimeIcons: {
     ALIGN_CENTER?: string,
     ALIGN_JUSTIFY?: string,
     ALIGN_LEFT?: string,

